### PR TITLE
Added a new email templating system with a login email

### PR DESCRIPTION
**Pull request by rootux**
# Implements
- Added new templating system - `ejs`
- Replaced the login email with organization name + url based on https://tedgoas.github.io/Cerberus/#responsive responsive emails

# Screenshots
The "Dev Root99 Organization" - would be replaced with Organization name
Screen Shot 2020-10-07 at 21 56 50

# Tophatting 🎩
- Tested that the app runs smoothly
- Tested that development login works
- Tested that email is being received and that link is working

### CODE REVIEW

File: api/services/EmailService/email.service.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
+const EmailTemplates = require('./email.templates');
 
 const mailgun = require('mailgun-js')({
   apiKey: process.env.MAILGUN_API_KEY,
   domain: process.env.MAILGUN_DOMAIN,
-  host: 'api.eu.mailgun.net',
```
Comment (rootux) [inline api/services/EmailService/email.service.js:5]: Hopefully that doesn't kill production @gustavlrsn

Comment (gustavlrsn) [reply api/services/EmailService/email.service.js:5]: I think it might kill production, maybe we can add a env variable `MAILGUN_HOST`?

**Comment by rootux** [comment]
@gustavlrsn Yes - you are right - we should have one defined. Is that the right way? Adding to .env.default ?

**Comment by gustavlrsn** [comment]
@

> @gustavlrsn Yes - you are right - we should have one defined. Is that the right way? Adding to .env.default ?

Yep! And I'll add it to the production environment

File: api/services/EmailService/login.template.ejs [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+
+        </table>
+        <!-- Email Body : END -->
+
+        <!-- Email Footer : BEGIN -->
+        <table align="center" role="presentation" cellspacing="0" cellpadding="0" border="0" width="100%" style="margin: auto;">
+            <tr>
+                <td style="padding: 20px; font-family: sans-serif; font-size: 12px; line-height: 15px; text-align: center; color: #ffffff;">
+                    <br><br>
+                    Dream platform - Plato<br><span class="unstyle-auto-detected-links"></span>
```
[the remark this answers is not in the document]
Comment (rootux) [reply api/services/EmailService/login.template.ejs:267]: Great idea - on it.

[the remark this answers is not in the document]
Comment (rootux) [reply api/services/EmailService/login.template.ejs:267]: Changed to the following:
Screen Shot 2020-10-11 at 14 09 06

**Pull request merged**